src/compiler/diagnosticMessages.json

```
@@ -4687,5 +4687,9 @@
     "Generate types for all packages without types": {
         "category": "Message",
         "code": 95068
+    },
+    "Types of construct signature is incompatible.": {

```

"is" should be "are"?

```suggestion
    "Types of construct signature are incompatible.": {
```